cmd/hccore/hccore_test.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
```

Comment:
test is failing because of the unused imports, just put an _ in front so travis doesn't complain.